src/bindings/io/doc/io_doc.c

```diff
@@ -0,0 +1,365 @@
+/**
+ * @file
+ *
+ * @brief IO Example binding
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <kdbhelper.h>
+#include <kdbio.h>
+#include <kdblogger.h>
+
+// Example I/O management library data structure
```

Comment:
What about using Doxygen comments everywhere? Or are all these structs specific to the doc plugin?